docs/pages/enterprise/sso.mdx

```diff
@@ -237,5 +239,44 @@ must be able to:
 If something is not working, we recommend to:
 
 - Double-check the host names, tokens and TCP ports in a connector definition.
-- Look into Teleport's audit log for claim mapping problems. It is usually stored on the
-  auth server in the `/var/lib/teleport/log` directory.ad
+- Review the below log information and the troubleshooting sections for the above guides.
+
+
+### Using the Web UI
+
+If you get "access denied" or other login errors, the number one place to check is the Audit
+Log. You can access it in the **Activity** tab of the Teleport Web UI.
+
+![Audit Log Entry for SSO Login error](../../img/sso/teleportauditlogssofailed.png)
+
+Example of a user being denied because the role `clusteradmin` wasn't set up:
+
+```json
+{
+  "code": "T1001W",
+  "error": "role clusteradmin is not found",
+  "event": "user.login",
+  "method": "oidc",
+  "success": false,
+  "time": "2019-06-15T19:38:07Z",
+  "uid": "cd9e45d0-b68c-43c3-87cf-73c4e0ec37e9"
+}
+```
+
+### No Servers show
+
+Servers will show if a user's roles include a matching server label that includes logins.
+Make sure the user's assigned roles have populated `logins` and match to the `node_labels`.
+In this example a user will have `ubuntu` and `debian` logins with a server that has `env: dev` tags.
```

Comment:
I think this sentence makes it sound like a user will inherit the `ubuntu` and `debian` logins for a Node that has the `env: dev` label. From [`TestCheckAccessToServer`](https://github.com/gravitational/teleport/blob/cc27d91a4585b0d744a7bec110260c335b0007fd/lib/services/role_test.go#L668), though (if I'm understanding this correctly), it seems like the Auth Service checks a user's logins against `allow.logins`, and checks `node_labels` against a Node's labels.

Reply:
updated